modules/integration/tests-common/clients/store/src/gen/java/org/wso2/am/integration/clients/store/api/v1/dto/PostRequestBodyDTO.java

```diff
@@ -0,0 +1,127 @@
+ *
+ * The version of the OpenAPI document: v2
+ * Contact: architecture@wso2.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package org.wso2.am.integration.clients.store.api.v1.dto;
+
+import java.util.Objects;
+import java.util.Arrays;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.io.IOException;
+import com.fasterxml.jackson.annotation.JsonCreator;
+/**
+* PostRequestBodyDTO
+*/
+
+public class PostRequestBodyDTO {
+        public static final String SERIALIZED_NAME_CONTENT = "content";
+        @SerializedName(SERIALIZED_NAME_CONTENT)
+            private String content;
+
+        public static final String SERIALIZED_NAME_CATEGORY = "category";
+        @SerializedName(SERIALIZED_NAME_CATEGORY)
+            private String category;
+
+
+        public PostRequestBodyDTO content(String content) {
+        
+        this.content = content;
+        return this;
+        }
+
+    /**
+        * Content of the comment 
+    * @return content
+    **/
+      @ApiModelProperty(example = "This is a comment", required = true, value = "Content of the comment ")
```

Comment:
This `PostRequestBodyDTO ` class has some bad formatting issues (not indented properly). Shall we fix those?

Reply:
Fixed in https://github.com/wso2/product-apim/pull/13178